### DISCUSSION TITLE
How to use both English/localized and original artist name in path formats?

### DISCUSSION THREAD

**Discussion by ali-ramadhan** [beetbox/beets#4105] [Q&A]
Just started using beets yesterday (moving over from Picard) and loving it so far!

I'm still trying to set up the "perfect" config and was wondering if anyone knew of a good way to use both the English/localized album artist name as well as the non-English/original name in path formats.

So something like

```
$localized_albumartist ($non_localized_albumartist)
Kaori Kobayashi (小林香織)
Ziad Rahbani (زياد الرحباني)
```

It seems like you can probably not set the `languages` option in the `config.yaml` so that `$albumartist` gets populated with the original artist name then use `$albumartist_sort` for the English/localized name (if they're different) since it's usually in English

```yaml
item_fields:
    bilingual_albumartist: u'%s' % (albumartist) if albumartist == albumartist_sort else u'%s (%s)' % (albumartist_sort, albumartist)
```

but this doesn't work well since `$albumartist_sort` is different sometimes (`Kobayashi, Kaori` or `String Cheese Incident, The` for example).

Sorry if I'm doing this awkwardly but any suggestions/recommendations would be appreciated!

**Comment by RollingStar** [comment]
This feels doable. It might be useful to write out explicitly how you expect certain situations to be handled. These could eventually become test cases. Ex:

EN name, no JP name
JP name, no EN name
both names
neither name
RTL text

I find this code a bit unreadable, would be better with some line breaks and more explicit if/elses, especially during planning/feedback.

    bilingual_albumartist: u'%s' % (albumartist) if albumartist == albumartist_sort else u'%s (%s)' % (albumartist_sort, albumartist)

You might need some way to define the canonical / native name. Not sure how MB does it.

https://musicbrainz.org/artist/9a197894-1a40-4650-89a2-928dcfcd1ef6/aliases

If MB doesn't provide it directly, could map country of origin / area to the language spoken there.
